nixos/modules/services/printing/cupsd.nix

```diff
@@ -291,6 +291,16 @@ in
 
     services.dbus.packages = [ cups.out ] ++ optional polkitEnabled cups-pk-helper;
 
+    # Allow asswordless printer admin for members of wheel group
+    security.polkit.extraConfig = mkIf polkitEnabled ''
+      polkit.addRule(function(action, subject) {
+          if (action.id == "org.opensuse.cupspkhelper.mechanism.all-edit" &&
+              subject.isInGroup("wheel")){
```

Comment:
Ubuntu's rule uses a special `lpadmin` group which actually would help the fact that `wheel` could actually not be an admin identity.
```
Identity=unix-group:lpadmin;unix-group:admin;unix-group:sudo
```